frontend/src/commponents/navBar.jsx

```diff
@@ -45,33 +32,33 @@ function NavBar(props){
                     </NavItem>
                     <div className="or">|</div>
                     <NavItem>
-                      <button className="but" onClick={props.onLogOut}>
+                      <button className="secButton" onClick={props.onLogOut}>
                         <span className="fa fa-sign-out"></span> Sign-out
                       </button>
                     </NavItem>
                   </React.Fragment>
                 )
-              :
-                (
-                  <React.Fragment>
-                    <NavItem>
-                      <button onClick={props.onClose} className="but">
-                        Log in
-                      </button>
-                    </NavItem>
-                    <NavItem>
-                      <button
-                        onClick={props.onRegister}
-                        className="btn btn-info"
-                      >
-                        Sign up
-                      </button>
-                    </NavItem>
-                  </React.Fragment>
+              :(
+                <>
+                  <button 
+                    onClick={props.onClose} 
+                    className="secButton"
+                  >
+                    Log in
+                  </button>
+
+                  <button
+                    onClick={props.onRegister}
```

Comment:
@tushhr this should be `props.onClose2`

Reply:
also can you try to change `onclose` and `onclose2` to `toggleModal` and `toggleModal2`

Reply:
I will do that in the next PR, fixing the Signup bug and updating forms..